bin/time-vdisp-cache

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python
+
+import numpy as np
+import argparse
+
+from desisim.templates import ELG
+from desisim.io import empty_metatable, read_basis_templates
+
+def make_elg(nobj=50, seed=123, fracvdisp=(0.1, 40), no_cache=False):
+    
+    rand = np.random.RandomState(seed)
+
+    if no_cache:
+        vdisp = 10**rand.normal(1.9, 0.15, nobj)
+    else:
+        nvdisp = int(np.min((np.fix(nobj * fracvdisp[0]), fracvdisp[1], 1)))
```

Comment:
I think you meant `np.max` rather than `np.min` here.  Your test is always testing the case of nvdisp=1 regardless of nobj.